### PR TITLE
Add dag for internal School Zone Beacons dataset

### PR DESCRIPTION
**Pull request by chiaberry**
I piggybacked a handful of typo fixing in the PR

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/13946

## Associated repo

https://github.com/cityofaustin/atd-knack-services/pull/122/

## Testing

**Steps to test:**

here is the dataset, it is an internal one, so you will have to log in to see it
https://datahub.austintexas.gov/Transportation-and-Mobility/School-Zone-Beacons-Internal/8whb-sg4d

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_knack_banner.py
Comment (chiaberry) [inline dags/atd_knack_banner.py:1]: again im piggybacking some fixes in here

File: dags/atd_kits_sig_stat_pub.py
Comment (chiaberry) [inline dags/atd_kits_sig_stat_pub.py:1]: fixing the misspelling of socrata

**Review by Charlie-Henry — APPROVED** [review]
Looks good 👍

File: dags/atd_kits_sig_stat_pub.py
Comment (johnclary) [reply dags/atd_kits_sig_stat_pub.py:1]: 🙈

File: dags/atd_knack_school_beacons.py [hunk 1]
```diff
@@ -68,6 +80,7 @@
     t1 = DockerOperator(
         task_id="atd_knack_school_beacons_to_postgrest",
         image=docker_image,
+        docker_conn_id="docker_default",
```
Comment (johnclary) [inline dags/atd_knack_school_beacons.py:83]: nice!

**Review by johnclary — APPROVED** [review]
thank you for all of these fixes! 🙌

**Review by mddilley — APPROVED** [review]
All looks great! I checked out corresponding PR and the open dataset. I see the new columns too. 🚀

**Pull request merged**